.vsts-ci/variables/environment.yml

```diff
@@ -0,0 +1,5 @@
+variables:
+  - name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
+    value: true
+  - name: DOTNET_CLI_TELEMETRY_OPTOUT
```

Comment:
I think if we end up removing the `build-images.yml`, this does't seem all that useful if it's just 2 variables.

Reply:
I know we might want to reuse this for the release, but we're not quite at that stage yet...